alf/algorithms/generator.py

```diff
@@ -104,12 +102,12 @@ def __init__(self,
             entropy_regularization (float): weight of entropy regularization
             kernel_sharpness (float): Used only for entropy_regularization > 0.
                 We calcualte the kernel in SVGD as:
-                    exp(-kernel_sharpness * reduce_mean((x-y)^2/width)),
-                where width is the elementwise moving average of (x-y)^2
+                    :math:`\exp(-kernel_sharpness * reduce_mean(\frac{(x-y)^2}{width}))`
```

Comment:
Use raw docstring if you have any slash in it, like r"""....""" (append 'r' in the front), otherwise some of slashes will be escaped.

Reply:
updated